docker-compose-light.yml

```diff
@@ -77,6 +77,7 @@ x-common-build: &common-build
     INCLUDE_CHROMIUM: ${INCLUDE_CHROMIUM:-false}
     INCLUDE_FIREFOX: ${INCLUDE_FIREFOX:-false}
     BUILD_TRANSLATIONS: ${BUILD_TRANSLATIONS:-false}
+    LOAD_EXAMPLES_DUCKDB: ${LOAD_EXAMPLES_DUCKDB:-true}
```

Comment:
Would it make sense to have the preferred platform as the value for the setting, rather than branded config keys? I.e. `EXAMPLES_DB = "duckdb"`?